plugins/scaffolder/src/components/fields/OwnerPicker/OwnerPicker.tsx

```diff
@@ -17,12 +17,21 @@ import React from 'react';
 import { EntityPicker } from '../EntityPicker/EntityPicker';
 import { FieldExtensionComponentProps } from '../../../extensions';
 
+/**
+ * The input props that can be specified under `ui:options` for the
+ * `OwnerPicker` field extension.
+ *
+ * @public
+ */
 export interface OwnerPickerUiOptions {
   allowedKinds?: string[];
 }
 
 /**
- * Owner Picker
+ * The underling component that is rendered in the form for the `OwnerPicker`
```

Comment:
```suggestion
 * The underlying component that is rendered in the form for the `OwnerPicker`
```

Signed-off-by: Fredrik Adelöw <freben@gmail.com>